Foundation/FileManager.swift

```diff
@@ -1289,9 +1684,13 @@ open class FileManager : NSObject {
             // Return true as the best guess.
             return true
         })
+#endif
     }
 
     private func _compareFiles(withFileSystemRepresentation file1Rep: UnsafePointer<Int8>, andFileSystemRepresentation file2Rep: UnsafePointer<Int8>, size: Int64, bufSize: Int) -> Bool {
+#if os(Windows)
```

Comment:
Maybe this needs to be written in terms of FileHandle?

Reply:
Yeah, I think that would be a nice cleanup.

Reply:
SR-9979